### PR TITLE
Ignore stylo crates in dependabot.yml

### PR DESCRIPTION
**Pull request by sagudev**
Their upgrades are done with companion PRs.

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [ ] These changes fix #___ (GitHub issue number if applicable)

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because ___

### CODE REVIEW

**Comment by sagudev** [comment]
I think something like this is also already in place for mozjs, but it was done via `@dependabot` commands.

**Review by mrobinson — APPROVED** [review]
Thank you! Feel free to land with this tiny nit:

File: .github/dependabot.yml [hunk 1]
```diff
@@ -8,3 +8,18 @@ updates:
   allow:
   - dependency-type: direct
   - dependency-type: indirect
+  ignore:
+  # ignore all stylo crates as their upgrades are cordinated via companion PRs
```
[the remark this answers is not in the document]
Comment (sagudev) [reply .github/dependabot.yml]: When editing files in GitHub UI there is no spell check.

[the remark this answers is not in the document]
Comment (mrobinson) [reply .github/dependabot.yml]: I totally understand. That's what a review is for. 😁

**Pull request merged**